config/samples/backends/nfs/backend.yaml

```diff
@@ -0,0 +1,28 @@
+# Sample using File/NFS as a glance backend
+# Requires a filepath ('/var/lib/glance/image/') is mounted on NFS share
+apiVersion: glance.openstack.org/v1beta1
+kind: Glance
+metadata:
+  name: glance
+spec:
+  serviceUser: glance
+  containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
+  customServiceConfig: |
+    [DEFAULT]
+    enabled_backends = default_backend:file
+    [glance_store]
+    default_backend = default_backend
+    [default_backend]
+    filesystem_store_datadir = /var/lib/glance/images/
```

Comment:
I guess we need here `extraMounts` w/ the nfs export we would like glance to be pointed to.
I'm going to update this example accordingly as long as I test it!

Reply:
Are you sure extraVolType should be Ceph here?

Reply:
no sorry, bad copy/paste, `extraVolType` is a label we can add to the extraMount to facilitate some logic in the operator. No relevant here, but definitely `s/ceph/nfs` is what we need

Reply:
other than this comment you had, in general it looks good!

Reply:
Done